fast/stages/1-resman/branch-networking.tf

```diff
@@ -98,22 +100,24 @@ module "branch-network-dev-folder" {
   name   = "Development"
   iam = {
     # read-write (apply) automation service accounts
-    (local.custom_roles.service_project_network_admin) = concat(
-      local.branch_optional_sa_lists.dp-dev,
-      local.branch_optional_sa_lists.gke-dev,
-      local.branch_optional_sa_lists.gcve-dev,
-      local.branch_optional_sa_lists.pf,
-      local.branch_optional_sa_lists.pf-dev,
-    )
+    (local.custom_roles.service_project_network_admin) = compact([
+      try(module.branch-dp-dev-sa[0].iam_email, null),
+      try(module.branch-gcve-dev-sa[0].iam_email, null),
+      try(module.branch-gke-dev-sa[0].iam_email, null),
+      try(module.branch-pf-sa[0].iam_email, null),
+      try(module.branch-pf-dev-sa[0].iam_email, null)
```

Comment:
I think `module.branch-pf-dev-sa[0]` should be `module.branch-pf-dev-sa` (same for pf-prod and so on) and the `try()` is hiding the error. 

I'd try to avoid all these `try()`s if possible (maybe [`one()` + splat](https://developer.hashicorp.com/terraform/language/functions/one#relationship-to-the-splat-operator) might be an option for the conditional ones)

Reply:
the `try`s are there to prevent issues during destroy AFAIR, good catch on the `[0]`

Reply:
removed `[0]`s